dev/com.ibm.ws.opentracing.1.x_fat/fat/src/com/ibm/ws/testing/opentracing/test/FATMPOpenTracing.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2020 IBM Corpo<ration and others.
+ * Copyright (c) 2017, 2022 IBM Corporation and others.
```

Comment:
`Corpo<ration` LOL

Reply:
Thanks, found a few instanced of that :)